README.md

```diff
@@ -2,5 +2,4 @@
 backend code 
 
 # Swagger UI 
-To get swagger UI display, add '../swagger-ui.html' into the url 
-for example: https://localhost:8080/swagger-ui.html
\ No newline at end of file
+localhost: http://localhost:9000/swagger-ui/index.html#/
```

Comment:
[Q]: Is this match with server's port?

Reply:
any port that has been configured in application.propertities